### ISSUE TITLE
How do i call the show method? Kindly help

### ISSUE THREAD

**Issue by patrickkabwe** [siemiatj/react-native-modal-dropdown#21]
![Screenshot 2021-03-05 at 6 25 25 AM](https://user-images.githubusercontent.com/64147010/110067159-c5540780-7d7b-11eb-9379-745de1c11cab.png)

Hi donni106,
Thanks for following up,
In the above image there is a show( ) method which triggers the dropdown Modal to show. 
My question was how can i call the show( ) method in a case where i am using a custom Icon to display the dropdown modal?
Thank you

**Comment by donni106** [comment]
can you elaborate a bit more please what you need to know

**Comment by smriti-spotnana** [comment]
I think they want to know how to call the show() method manually? - rather than lib calling it itself

**Comment by patrickkabwe** [comment]
@smriti-spotnana Any suggestion?

**Comment by smriti-spotnana** [comment]
Nope, I couldn't figure out myself ..

**Comment by donni106** [comment]
Please have a look in the examples, there you can find one for using `show()` (dropdown_5).

Code for examples with Expo: https://github.com/siemiatj/react-native-modal-dropdown/blob/master/expo-example/App.js
Code for examples without Expo: https://github.com/siemiatj/react-native-modal-dropdown/blob/master/example/index.js

You can also try the examples running on a phone or simulator.

**Comment by jayarajm** [comment]
Yes I can call show(), But the drop is not open. I hope the fullscreen overlay is coming but dropdown is not open.

**Comment by gorbs** [comment]
```javascript
const dropdownRef = useRef();

return (
<View>
<Button title="press" onPress={() => dropdownRef.current.show()} />
<ModalDropdown
  ref={dropdownRef}
  options={['aaa','bbb','ccc']}
/> 
</View>
)
```
